### PR TITLE
SMP-24 Add configurtion item skip

### PR DESCRIPTION
**Pull request by ern**
Example:

```
      <plugin>
        <groupId>org.sakaiproject.maven.plugins</groupId>
        <artifactId>sakai</artifactId>
        <configuration>
          <skip>true</skip>
        </configuration>
      </plugin>
```

### CODE REVIEW

File: sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployMojo.java [hunk 1]
```diff
@@ -78,6 +78,13 @@
 
 	private static Properties defaultLocationMap;
 	
+    /**
```
Comment (buckett) [inline sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployMojo.java]: Indentation looks wrong.

File: sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployExplodedMojo.java [hunk 2]
```diff
@@ -68,6 +68,13 @@
 
 	private static Properties defaultLocationMap;
 	
+    /**
```
Comment (buckett) [inline sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployExplodedMojo.java]: Indentation looks wrong.

File: sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployMojo.java (hunk 1 above)
Comment (ern) [inline sakai-plugin/src/main/java/org/sakaiproject/maven/plugin/component/ComponentDeployMojo.java]: Yeah eclipse gets it wrong every time it seems.

**Comment by buckett** [comment]
Magic, thanks.

**Pull request merged**